lib/iris/experimental/ugrid/mesh.py

```diff
@@ -122,25 +122,31 @@ def __init__(
         units=None,
         attributes=None,
         start_index=0,
-        src_dim=0,
+        location_axis=0,
     ):
         """
         Constructs a single connectivity.
 
         Args:
 
         * indices (numpy.ndarray or numpy.ma.core.MaskedArray or dask.array.Array):
-            The index values describing a topological relationship. Constructed
-            of 2 dimensions - the list of locations, and within each location:
-            the indices of the 'target locations' it relates to.
-            Use a :class:`numpy.ma.core.MaskedArray` if :attr:`src_location`
-            lengths vary - mask unused index 'slots' within each
-            :attr:`src_location`. Use a :class:`dask.array.Array` to keep
-            indices 'lazy'.
+            2D array giving the topological connection relationship between
+            ' :attr:`location` elements ' and ' :attr:`connected` elements '.
+            The :attr:`location_axis` dimension indexes over the
+            :attr:`location` dimension of the mesh - i.e. its length matches
+            the total number of :attr:`location` elements in the mesh. The
+            :attr:`connected_axis` dimension can be any length, corresponding
+            to the highest number of :attr:`connected` elements connected to a
+            :attr:`location` element. The array values are indices into the
+            :attr:`connected` dimension of the mesh. Use a
```

Comment:
> Use a MaskedArray if the number of 'connected' elements varies between 'location' elements - mask a 'location` element's unused index 'slots'.

This now reads rather oddly.
I'd suggest maybe re-ordering : "*If* the number of 'connected' elements varies between 'location' elements, *then* use a MaskedArray *and* mask *the* 'location` element's unused index 'slots'"
?

Reply:
7471929b585df6ff8718a75b2224b1027a5331d2